validator/sawtooth_validator/journal/block_manager.py

```diff
@@ -61,6 +61,13 @@ def put(self, branch):
                  self.pointer,
                  ctypes.py_object(branch))
 
+    def __contains__(self, item):
+        try:
+            next(self.get([item]))
+            return True
+        except StopIteration:
+            return False
+
```

Comment:
This is a somewhat non-intuitive implementation. Can you add a comment around the fact that it is depending on `get()` to return either exactly 1 or exactly 0 items?